packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -4,13 +4,17 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-alert-width: Specifies the width of the component.
+ * @prop --calcite-alert-background-color: Specifies the component's background color.
+ * @prop --calcite-alert-text-color: Specifies the component's text color.
+ * @prop --calcite-alert-border-radius: Specifies the component's border radius.
+ * @prop --calcite-alert-close-background-color-hover: Specifies the component's close button background color when hovered.
  */
 
 $border-style: 1px solid var(--calcite-color-border-3);
 
 :host {
-  --calcite-alert-edge-distance: theme("spacing.8");
-  --calcite-alert-dismiss-progress-background: var(--calcite-color-transparent-tint);
+  --calcite-internal-alert-edge-distance: theme("spacing.8");
```

Comment:
This should be alert-size not edge-distance

Reply:
`alert-size` might not be accurate. This represents the distance used to place the alert along the page.

Reply:
I updated this to `--calcite-internal-alert-edge-distance`